.env.local.default

```diff
@@ -56,6 +57,7 @@ SERVER_MODE=local
 SERVER_HOST=localhost
 SERVER_PORT=3030
 SERVER_URL=https://localhost:3030
+CROS_SERVER_PORT=3035
```

Comment:
Typo here - should be 'CORS_SERVER_PORT', as well as down in server-core/src/appconfig.ts where it's used.